src/Gadfly.jl

```diff
@@ -1105,6 +1105,22 @@ function display(d::REPLDisplay, ::MIME"application/pdf", p::Plot)
     open_file(filename)
 end
 
+# Display in Juno
+
+import Juno: Juno, @render, media, Media, Hiccup
```

Comment:
FYI @pfitzseb, something like in the Juno stack may have been the cause of the v0.5 upgrade precompilation error seen before. Not this specifically, but if you used the same conditional dependency workaround somewhere else, it would lead to t and be fixed by the fix in the fix found in the issue:

https://github.com/JunoLab/uber-juno/issues/15